modules/fb_instant_articles_display/src/EntityPropertyMapper.php

```diff
@@ -109,16 +109,27 @@ private function addHeaderFromProperties() {
       );
     }
 
+    // user token or static string for author name
+    $author = '';
```

Comment:
No need to define this is you change the if statement below. See this comment below: https://github.com/BurdaMagazinOrg/module-fb_instant_articles/pull/71/files#r67538644